components/raftstore-v2/src/batch/store.rs

```diff
@@ -616,12 +618,29 @@ impl<EK: KvEngine, ER: RaftEngine> StoreSystem<EK, ER> {
             let raft_clone = raft_engine.clone();
             let logger = self.logger.clone();
             let router = router.clone();
+            let registry = tablet_registry.clone();
             worker.spawn_interval_task(cfg.value().raft_engine_purge_interval.0, move || {
+                let limiter = Limiter::new(MAX_MANUAL_FLUSH_RATE);
                 let _guard = WithIoType::new(IoType::RewriteLog);
                 match raft_clone.manual_purge() {
-                    Ok(regions) => {
+                    Ok(mut regions) => {
+                        warn!(logger, "flushing oldest cf of regions {regions:?}");
+                        // Refresh the list at least every 60s.
+                        regions.truncate((MAX_MANUAL_FLUSH_RATE * 60.0) as usize);
```

Comment:
Here, `MAX_MANUAL_FLUSH_RATE` == 0.3 * 60.0 => 18 as default, an empirical value proved appropriate for `raftstore-v2`?

> Refresh the list at least every 60s.

And what's the meaning of this annotation? Might not clear for this.

Reply:
0.3 is the max speed, 0.3*60 is the maximum memtables we can flush within 60s.

Reply:
got.